client/include/get_command.h

```diff
@@ -13,9 +13,21 @@
 #include <string.h>
 #include <stdio.h>
 
-int login_client(char *command, int socket);
-int logout_client(char *command, int socket);
+/// Manage commands for client
+///
+/// /get_commands
+int unsubscribe_teams(char *av, int socket);
+int subscribe_teams(char *av, int socket);
 int send_message(char *av, int socket);
+int logout_client(char *av, int socket);
+int login_client(char *av, int socket);
+int use(char *av, int socket);
+int info(char *av, int socket);
+int list(char *av, int socket);
+int create(char *av, int socket);
+int messages(char *av, int socket);
+int user(char *av, int socket);
+int users(char *av, int socket);
```

Comment:
Documentation is needed to understand how they work and what the parameters are (av ?).